### ISSUE TITLE
Allow the removal of cached credentials

### ISSUE THREAD

**Issue by wellloaded** [Windscribe/Desktop-App#173]
Very simple: once you provide the credential (e.g. custom OPVN/Wireguard conf file), we have the option to store (cache) the credentials.
I don't see anywhere a function to remove/clear credentials.

If I can suggest; this would make sense being a right click option on the connection itself e.g. a menu with an option "clear cached credentials", so that this can be performed on a connection/configuration basis.

P.S. in the meantime, while this is considered, how do I remove this manually?

Thanks

**Comment by jaxu** [comment]
Absolutely, this should be added.  I've opened a ticket on our board.

In the meantime, you can remove the `customOvpnAuths=...` line from ~/.config/Windscribe/Windscribe2.conf

**Comment by wellloaded** [comment]
> Absolutely, this should be added. I've opened a ticket on our board.
> 
> In the meantime, you can remove the `customOvpnAuths=...` line from ~/.config/Windscribe/Windscribe2.conf

where do I find the Windscribe2.conf on windows?

**Comment by jaxu** [comment]
I'm sorry, I'm not sure why I assumed Linux.  On Windows, it's stored in the registry at `Computer\HKEY_CURRENT_USER\SOFTWARE\Windscribe\Windscribe2`.  You can remove the `customOvpnAuth` key.

**Comment by wellloaded** [comment]
Hum I don't appear to have the customOvpnAuths key , perhaps I should have mentioned that my custom connection is wireguard.

![RemoteDesktopManager_nufF0E5bch](https://github.com/user-attachments/assets/4dac2669-0a63-49a6-a3a0-26e2703fbb4d)

**Comment by jaxu** [comment]
I'm not sure I understand.  Custom WG connections do not prompt for credentials and do not save anything.

**Comment by wellloaded** [comment]
I think you are right.... I was initially using OpenVPN (like 6 months ago) I since switched to wireguard.
The reason of this odd question of mine is that my VPN provider has merged into a new brand, and we got ask to log in with the new credentials. I suppose your point is very valid, authentication as in username/password if ever is relevant to OpeVPN only.

Thanks for the clarification I did overlook at this one.
